test/js/data/feature_tree.test.js

```diff
@@ -68,7 +72,8 @@ test('featuretree point query', function(t) {
 
     ft.query({
         source: "mapbox.mapbox-streets-v5",
-        scale: 724.0773439350247,
+        scale: 1.4142135624,
+        tileSize: 512,
```

Comment:
`1.4142135624 * 512 = 724.0773439350247`

I split the number into two numbers in https://github.com/mapbox/mapbox-gl-js/pull/2052 because I thought it would be clearer and closer to how we treat conversions elsewhere.